src/ai/backend/manager/api/schema.graphql

```diff
@@ -1387,6 +1393,12 @@ input CreateKeyPairResourcePolicyInput {
   max_vfolder_count: Int @deprecated(reason: "Deprecated since 23.09.4")
   max_vfolder_size: BigInt @deprecated(reason: "Deprecated since 23.09.4")
   max_quota_scope_size: BigInt @deprecated(reason: "Deprecated since 23.09.4")
+
+  """Added in 24.03.4."""
+  max_pending_session_count: Int
+
+  """Added in 24.03.4."""
+  max_pending_session_resource_slots: JSONString
```

Comment:
```suggestion
```

Reply:
this is `input CreateKeyPairResourcePolicyInput`, not `UserResourcePolicy`. please check again